vector/build.gradle

```diff
@@ -1,6 +1,7 @@
 import com.android.build.OutputFile
 
 apply plugin: 'com.android.application'
+apply plugin: 'com.google.firebase.appdistribution'
```

Comment:
I hope this will not be a problem for F-Droid build, but since we already have non free `com.google.android.gms.oss-licenses-plugin` just below, I think this will be OK.

Reply:
if we find out it is a problem we can use a similar mechanism to the coverage task and [conditionally apply the plugin](https://github.com/vector-im/element-android/pull/6478/files#diff-f3c10f5faa6b2c6a9a08eb7516581fb5afea4f1e5382c9b561666b8a6f4b6826L12)